24.md

```diff
@@ -17,6 +17,7 @@ These are extra fields not specified in NIP-01 that may be present in the string
   - `website`: a web URL related in any way to the event author.
   - `banner`: an URL to a wide (~1024x768) picture to be optionally displayed in the background of a profile screen.
   - `bot`: a boolean to clarify that the content is entirely or partially the result of automation, such as with chatbots or newsfeeds.
+  - `birth`: an array representing the author's birth date. The format is [day, month, year (optional)]. If the year is omitted, only the day and month should be considered.
```

Comment:
I suggest the word `birthday` instead of `birth`

```suggestion
  - `birthday`: an array representing the author's birth date. The format is [day, month, year (optional)]. If the year is omitted, only the day and month should be considered.
```